custom/templates/DefaultRevamp/members/members.tpl

```diff
@@ -0,0 +1,213 @@
+                            {/foreach}
+                        </select>
+                    </div>
+                </div>
+            </div>
+            <div class="ui fluid card">
+                <div class="content">
+                    <h4 class="ui header">New members</h4>
+                    <div class="description">
+                        <div class="ui four column grid" id="new-members-grid">
+                            {foreach from=$NEW_MEMBERS_VALUE item=member}
+                                <div class="column">
+                                    <a href="{$member->getProfileUrl()}" data-toggle="popup" data-poload="{$USER_INFO_URL}{$member->data()->id}">
+                                        <img class="ui circular image" src="{$member->getAvatar()}" alt="{$member->getDisplayname()}">
+                                    </a>
+                                </div>
+                            {/foreach}
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="ui column">
+            <div class="ui stackable equal width left aligned grid segment" style="margin-top: 0">
+                {if $VIEWING_LIST == "group"}
+                    <div class="ui column">
+                        <h3>{$VIEWING_GROUP->name}</h3>
+                        <div>
+                            <ul id="member_list_group_{$VIEWING_GROUP->id}" class="ui list large selection" style="margin-left: -10px;">
+                            </ul>
+                        </div>
+                    </div>
+                {else}
+                    {foreach from=$MEMBER_LISTS_VIEWING item=list}
+                        <div class="ui column">
+                            <h3>{$list->getFriendlyName()}</h3>
+                            <div>
+                                <ul id="member_list_{$list->getName()}" class="ui list large selection" style="margin-left: -10px;">
+                                </ul>
+                                {if $VIEWING_LIST == "overview"}
+                                    <a class="fluid ui grey basic button" href="{$list->url()}">{$VIEW_ALL}</a>
+                                {/if}
+                            </div>
+                        </div>
+                    {/foreach}
+                {/if}
+            </div>
+        </div>
+    </div>
+</div>
+
+<script type="text/javascript">
+    const viewGroup = (e) => {
+        window.location.href = '{$VIEW_GROUP_URL}' + e.value;
+    }
+
+    const renderList = (name) => {
+        return function () {
+            const xhr = new XMLHttpRequest();
+            xhr.withCredentials = false;
+            xhr.open('GET', '{$QUERIES_URL}'.replace(
+                {literal}
+                '{{list}}',
+                {/literal}
+                name
+            ));
```

Comment:
Can we please stop using these ancient methods of requesting endpoints? Seeing as we don't support IE anymore we can use the fetch api, which results in way cleaner code, especially if we use async/await

Reply:
Thank you! Always forget that is in the browser 🤦🏼‍♂️